app/components/x-form/datepicker.js

```diff
@@ -0,0 +1,22 @@
+import Date from 's1ack/components/x-form/date';
+import { get, set} from '@ember/object';
+
+export default Date.extend({
+  inputType: 'text',
+  didInsertElement(){
+    this._super(...arguments);
+
+    set(this, 'pikaday', new Pikaday({
+           field: this.element.querySelector('input'), // or we can use JQuery 
+           format: 'YYYY-MM-DD',
```

Comment:
format might be defined as property of the component (so any user of the component can customise that)